moonstreamapi/moonstreamapi/settings.py

```diff
@@ -309,6 +309,8 @@
     "user_id",
     "address",
     "blockchain",
+    "customer_id",
```

Comment:
What about journal entities changes (at spire). We added here reserved tags, but not modified the source. Why just not to use `required_fields` https://spire.bugout.dev/journals/docs#tag/entities/operation/create_journal_entity__journal_id__entities_post
?

Reply:
Old one remain same. New one will have new tags